packs/ansible/README.md

```diff
@@ -0,0 +1,61 @@
+# <img src="http://www.ansible.com/favicon.ico" width="32px" valign="-3px"/> Ansible Integration Pack
+This pack provides [Ansible](http://www.ansible.com/) integration to perform remote operations on both local and remote machines.
+After [pack installation](http://docs.stackstorm.com/packs.html#getting-a-pack) all ansible executable files are available in pack virtualenv and ready to use.
+
+## Actions
+* `command` - Run single [Ad-Hoc command](http://docs.ansible.com/intro_adhoc.html). It has all the regular parameters of `ansible` executable.
+* `command_local` - Perform single ansible Ad-Hoc command (module) locally.
+* `playbook` - Action to run [Ansible Playbook](http://docs.ansible.com/playbooks.html) (`ansible-playbook` executable).
+* `vault.encrypt` - Encrypt ansible data files (playbooks, vars, roles, etc) with password (`vault` executable).
+* `vault.decrypt` - Decrypt ansible data files (playbooks, vars, roles, etc) with password (`vault` executable).
+* `galaxy.install` - Install role from [Ansible Galaxy](http://docs.ansible.com/galaxy.html) - hub of [community developed roles](https://galaxy.ansible.com/) (`ansible-galaxy`).
+* `galaxy.list` - List installed from Ansible Galaxy roles (`ansible-galaxy` executable).
+* `galaxy.remove` - Remove the role installed from Ansible Galaxy (`ansible-galaxy` executable).
+
+## Examples
+#### `ansible.command` examples
+```sh
+# run ansible command with optional verbose parameter
+st2 run ansible.command args='hostname -i' verbose=vv
```

Comment:
You're right, it's a mistake.
Correct version is: https://github.com/armab/st2-ansible-vagrant/blob/master/ansible.sh#L55